.travis.yml

```diff
@@ -15,9 +15,15 @@ jobs:
     script:
     - npm run start-ganache &
     - npm run test
+  - stage: Pull Contracts from npm
```

Comment:
Why do we need to pull the contracts from npm before running the next stage? Are there dependencies?

Reply:
We persist the previous deployment addresses in the npm repo. So we pull it down the latest release and when truffle runs the migrations it can see if the network is up date. The Migrations.json is the especially important

Reply:
For example know we have the rinkeby addresses and we deploy to kovan we want to append the new addresses to contract deployment files